src/security.cs

```diff
@@ -337,178 +337,220 @@ interface KeysAccessible {
 		IntPtr WhenPasscodeSetThisDeviceOnly { get; }
 	}
 
-	[Static][Internal]
+	[Static]
 	interface SecAttributeKey {
-		[Mac (10,9)]
+		[iOS (4,0), Mac (10,9), TV (9,0), Watch (2,0)]
```

Comment:
The minimum iOS version we support is **iOS 6** so a `[iOS (4,0)]` is unnecessary (:

Also any API in tvOS without an attribute is **implicitly 9,0**. Similarly, any watchOS API without an attribute is **implicitly 2,0**. Therefore we try not to include `TV (9,0), Watch (2,0)` in our binding files (because it's redundant).

Reply:
Comment applies to all availability attributes you added (:

Reply:
I guess that the implicit version Mac is 10,7 and therefore all [Mac (10,6)] should also be removed.